bundles/org.openhab.binding.philipsair/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,185 @@
+		<state pattern="%s" />
+	</channel-type>
+	<channel-type id="ddp">
+		<item-type>String</item-type>
+		<label>Displayed index</label>
+		<description>Display PM2.5 index or Allergen Index</description>
+		<state readOnly="false" pattern="%s">
+			<options>
+				<option value="1">PM2.5</option>
+				<option value="0">Allergen</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="aqil">
+		<item-type>Number</item-type>
+		<label>LED light level</label>
+		<description>Level 100, 75, 50, 25, 0</description>
+		<state pattern="%d" min="0" max="100" step="25" />
+	</channel-type>
+	<channel-type id="cl">
+		<item-type>Switch</item-type>
+		<label>Child lock</label>
+		<description>Child lock enabled/disabled</description>
+	</channel-type>
+
+	<channel-type id="dtrs">
+		<item-type>Number:Time</item-type>
+		<label>Timer Left</label>
+		<description>Timer minutes lefts</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="dt">
+		<item-type>Number</item-type>
+		<label>Timer</label>
+		<description>Timer in hours (0-5)</description>
+		<state min="0" max="5" pattern="%d" step="1" />
+	</channel-type>
+
+	<channel-type id="pm25">
+		<item-type>Number:Density</item-type>
+		<label>PM2.5</label>
+		<description>PM 2.5 particles pollution level</description>
+		<state readOnly="true" pattern="%d" min="0" max="5000" />
+	</channel-type>
+	<channel-type id="iaql">
+		<item-type>Number</item-type>
+		<label>IAQL</label>
+		<description>Allergen index</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="aqit">
+		<item-type>Number</item-type>
+		<label>AQIT</label>
+		<description>AQI</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="err">
+		<item-type>String</item-type>
+		<label>Error Message</label>
+
+		<state readOnly="true" pattern="%s">
+			<options>
+				<option value="0">No error</option>
+				<option value="49408">No water</option>
+				<option value="32768">Water tank open</option>
+				<option value="49155">Clean pre-filter</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fltsts0">
+		<item-type>Number</item-type>
+		<label>Pre-filter</label>
+		<description>Pre-filter estimated clean up</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="wicksts">
+		<item-type>Number</item-type>
+		<label>Wick</label>
+		<description>Wick filter estimated lifetime</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="fltsts2">
+		<item-type>Number</item-type>
+		<label>Carbon</label>
+		<description>Active carbon filter estimated lifetime</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="fltsts1">
+		<item-type>Number</item-type>
+		<label>HEPA Filter</label>
+		<description>HEPA filter estimated lifetime</description>
+		<state readOnly="true" pattern="%d" />

```

Comment:
same question here

Reply:
The max value is hard to determine as it vary from one model to another as they use different filters and it is not available  in the official specification.